admin-tools/src/webapp/vm/user/chef_users_list.vm

```diff
@@ -11,7 +11,7 @@
 		$tlang.getString("uselis.users")
 	</h3>
 
-		#if ($alertMessage)<div class="alertMessage">$tlang.getString("useconrem.alert") $validator.escapeHtml($alertMessage)</div>#end
+		#if ($alertMessage)<div class="sak-banner-error">$tlang.getString("useconrem.alert") $validator.escapeHtml($alertMessage)</div>#end
 		#if ($successMessage)<div class="success">$validator.escapeHtml($successMessage)</div>#end
```

Comment:
Missed a success message? Are there others?

Reply:
Good catch, sorry. I missed two `class="success"`, and I've changed them to `class="sak-banner-success"`